rgdps/api/__init__.py

```diff
@@ -12,7 +12,7 @@
 from fastapi.responses import JSONResponse
 from fastapi.responses import Response
 from fastapi_limiter import FastAPILimiter
-from meilisearch_python_async import Client as MeiliClient
+from meilisearch_python_sdk import Client as MeiliClient
```

Comment:
This should be `AsyncClient` afaik (alongside all other instances of this import).

Reply:
your fault for not using asyncclient

Reply:
the client in `meilisearch_python_async ` was async